modules/aggregation-layers/src/utils/bin-sorter.js

```diff
@@ -64,38 +69,51 @@ export default class BinSorter {
    * @return {Array} array of new value range
    */
   getValueRange([lower, upper]) {
-    const len = this.sortedBins.length;
+    const len = this.aggregatedBins.length;
     if (!len) {
       return [0, 0];
     }
     const lowerIdx = Math.ceil((lower / 100) * (len - 1));
     const upperIdx = Math.floor((upper / 100) * (len - 1));
 
-    return [this.sortedBins[lowerIdx].value, this.sortedBins[upperIdx].value];
-  }
-
-  /**
-   * Get ths max count of all bins
-   * @return {Number | Boolean} max count
-   */
-  getMaxCount() {
-    let maxCount = 0;
-    this.sortedBins.forEach(x => (maxCount = maxCount > x.counts ? maxCount : x.counts));
-    return maxCount;
+    return [this.aggregatedBins[lowerIdx].value, this.aggregatedBins[upperIdx].value];
   }
 
   /**
    * Get a mapping from cell/hexagon index to sorted bin
    * This is used to retrieve bin value for color calculation
-   * @return {Object} bin index to sortedBins
+   * @return {Object} bin index to aggregatedBins
    */
   getBinMap() {
```

Comment:
Is this a public method?

Reply:
@heshan0131 is going to mark private/public methods in this file.